### PR TITLE
chore: Update light mode colors

### PR DESCRIPTION
**Pull request by Rperry2174**
we're getting closer and closer to lightmode being release-able
image

things that still need some love
- single timeline: yellow is hard to see on white background
- timeline tooltip: ugly border
- comparison timeline: both pink and blue should be different colors probably

### CODE REVIEW

File: webapp/javascript/components/Sidebar.tsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ import {
   uncollapseSidebar,
   recalculateSidebar,
 } from '@webapp/redux/reducers/ui';
+// import useColorMode from '@webapp/hooks/colorMode.hook';
```
Comment (Rperry2174) [inline webapp/javascript/components/Sidebar.tsx:29]: leaving this in so I can turn on and off color mode when testing

File: webapp/javascript/components/Sidebar.tsx [hunk 2]
```diff
@@ -42,6 +43,7 @@ function signOut() {
 
 export function SidebarComponent() {
   const collapsed = useAppSelector(selectSidebarCollapsed);
+  // const { changeColorMode, colorMode } = useColorMode();
```
Comment (Rperry2174) [inline webapp/javascript/components/Sidebar.tsx:46]: same here

**Comment by petethepig** [comment]
Looks so good!

**Review by petethepig — APPROVED** [review]
Great colors!

**Pull request merged**